content/design-systems-for-developers/react/en/test.md

```diff
@@ -60,64 +62,29 @@ npx chromatic --project-token=<project-token>
 ```
 
 ![Result of our first Chromatic build](/design-systems-for-developers/chromatic-first-build.png)
+-->
 
-Chromatic captured a baseline image of every story! Subsequent test runs will capture new images and compare them against these baselines. See how that works by tweaking a UI component and saving it. Go to the global styles (`src/shared/styles.js`) and increase the font-size.
+When the pull request was created, asides from publishing Storybook, Chromatic also captured a set of images for the changes that we made!
```

Comment:
```suggestion
When the pull request was created, Chromatic published Storybook then captured a set of images for the changes that we made!
```

Reply:
Suggested wording applied per feedback. Marking it as resolved.